packages/css-property-parser/lib/Parser.re

```diff
@@ -141,7 +141,7 @@ and calc_value = [%value.rec
 and cf_final_image = [%value.rec "<image> | <color>"]
 and cf_mixing_image = [%value.rec "[ <extended-percentage> ]? && <image>"]
 and class_selector = [%value.rec "'.' <ident-token>"]
-and clip_source = [%value.rec "<url>"]
+and clip_source = [%value.rec "<url-with-interp>"]
```

Comment:
i usually name things with interp "extended", would you prefer to keep -with-interp or -extended? 

both looks good to me, but want to align here

Reply:
i think we can use the -no-interp suffix naming? like we did for transition and animation